kmsencryption/lib.py

```diff
@@ -26,14 +28,20 @@ def decrypt_value(data, prefix, key_provider):
         data = data[len(prefix):]
 
     raw_data = base64.b64decode(data)
-    decrypted_plaintext, decryptor_header = aws_encryption_sdk.decrypt(
+    client = aws_encryption_sdk.EncryptionSDKClient(
+        commitment_policy=aws_encryption_sdk.CommitmentPolicy.FORBID_ENCRYPT_ALLOW_DECRYPT,
+    )
+    decrypted_plaintext, decryptor_header = client.decrypt(
         source=raw_data,
         key_provider=key_provider)
     return decrypted_plaintext
 
 
 def encrypt_value(data, prefix, key_provider):
-    ciphertext, encryptor_header = aws_encryption_sdk.encrypt(
+    client = aws_encryption_sdk.EncryptionSDKClient(
+        commitment_policy=aws_encryption_sdk.CommitmentPolicy.FORBID_ENCRYPT_ALLOW_DECRYPT,
```

Comment:
That's probably tested and correct, but it looks strange to use `FORBID_ENCRYPT_ALLOW_DECRYPT` policy in encryption func 👀 . Maybe you can add some comment?

Reply:
This is for the "commitment policy", which is something that was added in v1.7 of the `aws-encryption-sdk` library. This setting retains the old behavior.

https://docs.aws.amazon.com/encryption-sdk/latest/developer-guide/migrate-commitment-policy.html